interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,451 @@
+                <description>Incoming packets that are not meant for the local node, but which should be forwarded to another node, are dropped</description>
+              </valueHelp>
+              <valueHelp>
+                <format>kernel</format>
+                <description>Incoming packets are always sent to the TUN/TAP device, even if the packets are not for the local node. This is less efficient, but allows the kernel to apply its routing and firewall rules on them, and can also help debugging</description>
+              </valueHelp>
+              <constraint>
+                <regex>(off|kernel)</regex>
+              </constraint>
+            </properties>
+            <defaultValue>internal</defaultValue>
+          </leafNode>
+          <leafNode name="graph-dump-file">
+            <properties>
+              <help>Dump network graph files, which can be read with graphviz</help>
+            </properties>
+          </leafNode>
+          <leafNode name="resolve-hostname">
+            <properties>
+              <help>Resolve IP addresses (real and VPN)</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="iff-One-Queue">
+            <properties>
+              <help>Set IFF_ONE_QUEUE flag on TUN/TAP devices</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="key-expire">
+            <properties>
+              <help>Key expiration time(seconds,default:3600)</help>
+            </properties>
+            <defaultValue>3600</defaultValue>
+          </leafNode>
+          <leafNode name="local-discovery">
+            <properties>
+              <help>Enable Local Discovery</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="mac-expire">
+            <properties>
+              <help>MAC expiration time(seconds,default:600)</help>
+            </properties>
+            <defaultValue>600</defaultValue>
+          </leafNode>
+          <leafNode name="max-timeout">
+            <properties>
+              <help>Max Timeout(seconds,default:900)</help>
+            </properties>
+            <defaultValue>900</defaultValue>
+          </leafNode>
+          <leafNode name="ping-interval">
+            <properties>
+              <help>Ping Interval(seconds,default:60)</help>
+            </properties>
+            <defaultValue>60</defaultValue>
+          </leafNode>
+          <leafNode name="ping-timeout">
+            <properties>
+              <help>Ping Timeout(seconds,default:5)</help>
+            </properties>
+            <defaultValue>5</defaultValue>
+          </leafNode>
+          <leafNode name="priority-inheritance">
+            <properties>
+              <help>Enable Priority Inheritance</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="private-keyfile">
```

Comment:
We should probably use the PKI instrastructure now that OpenVPN etc. are using it already. I mean, `set pki ...`. I'd be curious what @sever-sever, @c-po  and others think about that.